app/src/main/assets/debug/SketchApplication.java

```diff
@@ -20,33 +17,19 @@ public static Context getContext() {
     @Override
     public void onCreate() {
         mApplicationContext = getApplicationContext();
-        this.uncaughtExceptionHandler = Thread.getDefaultUncaughtExceptionHandler();
 
         Thread.setDefaultUncaughtExceptionHandler(
-            new Thread.UncaughtExceptionHandler() {
-                @Override
-                public void uncaughtException(Thread thread, Throwable throwable) {
-                    Intent intent = new Intent(getApplicationContext(), DebugActivity.class);
-                    intent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK);
-                    intent.putExtra("error", Log.getStackTraceString(throwable));
-
-                    PendingIntent pendingIntent =
-                        PendingIntent.getActivity(
-                            getApplicationContext(),
-                            11111,
-                            intent,
-                            PendingIntent.FLAG_ONE_SHOT
-                        );
-
-                    AlarmManager am = (AlarmManager) getSystemService(Context.ALARM_SERVICE);
-                    am.set(AlarmManager.ELAPSED_REALTIME_WAKEUP, 1000, pendingIntent);
-
-                    Process.killProcess(Process.myPid());
-                    System.exit(1);
-
-                    uncaughtExceptionHandler.uncaughtException(thread, throwable);
-                }
-            });
+                new Thread.UncaughtExceptionHandler() {
+                    @Override
+                    public void uncaughtException(Thread thread, Throwable throwable) {
+                        Intent intent = new Intent(getApplicationContext(), DebugActivity.class);
+                        intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
+                        intent.putExtra("stacktrace", Log.getStackTraceString(throwable));
```

Comment:
This will break existing projects with a custom crash handler.

Reply:
explain

Reply:
They rely on the extra being named `error`.